osquery/tables/system/darwin/asl_utils.cpp

```diff
@@ -0,0 +1,108 @@
+#include <boost/algorithm/string/replace.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include "asl_utils.h"
+
+namespace ba = boost::algorithm;
+
+namespace osquery {
+namespace tables {
+
+bool isNumeric(ColumnType coltype) {
```

Comment:
Just wondering, it makes sense to me why we would suggest to the compiler to inline this with `inline`, but why use `static`?